samples: matter: power-optimize lock (#5873)
nrfconnect/sdk-nrf: samples/matter/lock/README.rst

.. include:: /includes/sample_fem_support.txt

Low-power build
===============

> You can configure the sample to consume less power, using the ``-DOVERLAY_CONFIG="overlay-low_power.conf"`` build flag during the build process.
> This flag will enable the Thread's Sleepy End Device mode and disable debug features such as the UART console or the **LED 1** usage.
> 
> See :ref:`cmake_options` for instructions on how to add this option to your build.

Reviewer: ```suggestion
You can configure the sample to consume less power by using the low-power build, which enables Thread's Sleepy End Device mode and disables debug features, such as the UART console or the **LED 1** usage.

To trigger the low-power build, use the ``-DOVERLAY_CONFIG="overlay-low_power.conf"`` option when building the sample.
See :ref:`cmake_options` for instructions on how to add this option to your build.
```
